sequential-convoy/manifest.json

```
@@ -0,0 +1,60 @@
+{
+    "title": "Service Bus Peek Lock Pattern - Renew lock token",
+    "description": "Get new message from a Service Bus queue and lock it for processing, renewing the lock token until processing is completed. The message will be deleted from the queue after the processing is completed. This pattern is useful when you want to ensure that the message is processed only once and not lost in case of a failure during processing.",
+    "prerequisites": "Before you can use this template, you need to configure a Service Bus namespace and create a queue. For more information about Service Bus see [Create a new queue on Service Bus](https://learn.microsoft.com/en-us/azure/service-bus-messaging/service-bus-quickstart-portal)",
+    "skus": [
+        "standard"
+    ],
+    "kinds": [
+        "stateful",
+        "stateless"
+    ],
+    "details": {
+        "By": "Microsoft",
+        "Type": "Workflow",
+        "Trigger": "Event",    
+        "Category": "Design Patterns"
+    },
+    "artifacts": [
+        {
+            "type": "workflow",
+            "file": "workflow.json"
+        }
+    ],
+    "images": {
+        "light": "snapshot_light",
+        "dark": "snapshot_dark"
+    },
+    "parameters": [
+        {
+            "name": "queueName_#workflowname#",
+            "displayName": "Queue Name",

```

- Use sentence casing for parameter names, like the other templates.

```suggestion
            "displayName": "Queue name",

```